### ISSUE TITLE
UX Study - write up round 1 results

### ISSUE THREAD

**Issue by GenevieveHaliburton** [HumanCellAtlas/query-service#115]
Synthesize results from Query Service UX Study round 1 (internal developers) https://docs.google.com/document/d/1GkHRB5Tf1QpD-MEbP1gni2fetIndeV50NA73aIt-U-s/edit?usp=sharing

Blocked by completing interviews, last one scheduled for May 13th

**Comment by GenevieveHaliburton** [comment]
Results for interviews. Diary study was postponed.

Raw observations spreadsheet: https://docs.google.com/spreadsheets/d/1I5WZK9-Op06F-f4MKlGLA1Tfq-YkTTtg1LiWJzaSufs/edit#gid=0

Findings deck: https://docs.google.com/presentation/d/1FW9QNz623xVIJVgEsDj9snnM8u9EA8rH1q80PCPEcEg/edit#slide=id.g59a472010c_0_153
